src/styles/libraries/bootstrap-overrides/_grid.scss

```diff
@@ -19,7 +19,7 @@
 .col,
 [class^="col-"],
 [class*=" col-"] {
-  padding: $grid-gutter-width / 2;
+  padding: 0 $grid-gutter-width / 2;
```

Comment:
same here, i believe we get this for free

Reply:
same question